lib/amazon_product_api/search_response.rb

```diff
@@ -19,6 +24,14 @@ def items(item_class: SearchItem)
       item_hashes.map { |hash| item_class.new **item_attrs_from(hash) }
     end
 
+    def success?
+      SUCCESS_CODE_RANGE.include?(@code) || [FOUND_CODE, NOT_MODIFIED].include?(@code)
```

Comment:
Just a 💭 : Is it worth defining another constant like `SUCCESS_CODES = SUCCESS_CODE_RANGE.to_a + [FOUND_CODE, NOT_MODIFIED]` and then doing `SUCCESS_CODES.include?(@code)`